docs/src/content/docs/zh-cn/getting-started.mdx

```diff
@@ -45,24 +46,69 @@ yarn create astro --template starlight
 [在 StackBlitz 上打开模板](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics)。
 :::
 
-## 使用 Starlight 创建内容
+### 启动开发服务器
 
-Starlight 已经准备好让你添加新的内容，或者将你现有的文件带过来！
+在本地工作时，Astro的开发服务器允许你预览你的工作，并在你进行更改时自动刷新你的浏览器。
```

Comment:
In the English version of this paragraph, the text “Astro’s development server” links to the Astro docs: https://docs.astro.build/zh-cn/reference/cli-reference/#astro-dev

Can we add the link here too?

Reply:
Ouch, careless. I have updated it.